game_eggs/steamcmd_servers/scpsl/exiled/egg-s-c-p--s-l--exiled.json

```diff
@@ -0,0 +1,77 @@
+{
+    "_comment": "DO NOT EDIT: FILE GENERATED AUTOMATICALLY BY PTERODACTYL PANEL - PTERODACTYL.IO",
+    "meta": {
+        "version": "PTDL_v1",
+        "update_url": null
+    },
+    "exported_at": "2021-11-14T10:56:47+01:00",
+    "name": "SCP:SL - Exiled",
+    "author": "info@goover.de",
+    "description": "Egg for SCP: Secret Laboratory Dedicated Linux Server with Exiled Plugin Framework",
+    "features": null,
+    "images": [
+        "ghcr.io\/parkervcp\/yolks:mono_latest"
+    ],
+    "file_denylist": [],
+    "startup": ".\/LocalAdmin {{SERVER_PORT}}",
+    "config": {
+        "files": "{}",
+        "startup": "{\r\n    \"done\": \"Waiting for players..\"\r\n}",
+        "logs": "{\r\n    \"custom\": false,\r\n    \"location\": \"logs\/latest.log\"\r\n}",
+        "stop": "^C"
```

Comment:
This game has its own stop command, `EXIT` that would gracefully stop it

Reply:
done